src/uuidToBytes.js

```diff
@@ -0,0 +1,39 @@
+import validate from './validate.js';
+
+function uuidToBytes(uuid) {
```

Comment:
The current UUID proposal at https://github.com/tc39/proposal-uuid#uuid-api always uses all-uppercase `UUID` in method naming. The reasoning was, that acronyms are usually not camel-cased in the JavaScript language, e.g. [`decodeURIComponent()`](https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/decodeURIComponent), [`url.toJSON()`](https://developer.mozilla.org/de/docs/Web/API/URL/toJSON), …

I'd personally prefer to use this kind of naming here as well:

```
UUIDToBytes()
bytesToUUID()
```

If you feel strongly against it I can also live with the current state.

Reply:
I actually dislike acronyms in method names for exactly this reason.  It's bad enough when the acronym appears in the middle of a name (is it `innerHtml` or `innerHTML`?  I can never remember.  😢), but when it appears at the beginning of function or method name - where you really don't expect uppercase - it's visually jarring.    `UUIDToBytes` just looks weird.  Note, too, that nowhere in Node's built-in API (2500+ method names) is there a method that starts with an uppercase letter.

Maybe the best thing to do here is just emulate the JSON API and rename these to `parse` (uuidToBytes) and `stringify` (bytesToUuid).  Thoughts?

Reply:
Re parse/stringify: Is a byte array the canonical representation of a UUID so that especially `parse` is self-explaining? The only other representation that comes to my mind would be something like `int128` but since something like that isn't part of the JavaScript language anyways I guess it's safe to assume that a byte array is the only sane in-memory representation of a UUID…

Now what about the broader API surface (from the `v9` branch)?

```
UUID.validate()
UUID.version()
UUID.parse()
UUID.stringify()
```

Together with a namespace these all seem to be pretty straightforward and `UUID.bytesToUuid()` would indeed be inconsistent.

It only conflicts a little bit with the current [`randomUUID()` proposal](https://github.com/tc39/proposal-uuid#uuid-api) but since that proposal is in a very early stage I'm not sure if it makes sense at all to have that influence our API evolution within this module.

So yeah, `stringify`/`parse` would be fine for me 👍 .

Reply:
> Is a byte array the canonical representation of a UUID so that especially `parse` is self-explaining?

Yes. 

That's what I used as the backing data structure for the `UUID` class [I proposed](https://codepen.io/broofa/pen/KYOvWE?editors=0010) and I found it worked well.  Also, it's what this module used in the [old `parse` and `unparse` API](https://github.com/uuidjs/uuid/tree/v1.4.4#uuidparseid-buffer-offset) it used to export (as `Buffer` instances).

The use of `Buffer` in that old API was mostly because a) this module initially targeted Node and b) typed arrays weren't a thing yet, fwiw.  Clearly that's not the right choice anymore, so I think the only real decision here is should we go with `Array` or `Uint8Array`?  I'm inclined to say the latter.  I.e. We can have `unparse()` accept any array-like[16] input, but `parse()` would only ever return a `Uint8Array`.

Reply:
Sounds reasonable.

Reply:
BTW that's deno's choice as well: https://github.com/denoland/deno/blob/d615f0ff82f5b2b530940a21a61e5b4b08b1d1f4/std/uuid/_common.ts#L40-L47